include/deal.II/base/partitioner.h

```diff
@@ -113,8 +115,15 @@ namespace Utilities
       /**
        * Allows to set the ghost indices after the constructor has been
        * called.
+       *
+       * The optional parameter @p larger_ghost_index_set allows defining an
+       * indirect addressing into a larger set of ghost indices. This setup is
+       * useful if a distributed vector is based on that larger ghost index
+       * set but only a tighter subset should be communicated according to
+       * @p larger_ghost_index_set.
```

Comment:
according to ghost_indices? Meaning that `ghost_indices \in larger_ghost_index_set` ?